MOxUnit/@MOxUnitTestResult/report_xml.m

```diff
@@ -0,0 +1,164 @@
+function x = report_xml(obj,c,t,dur,reason)
+% Add an XML report of a single test case result
+% (error, failure, skip, or success)
+%
+% report_xml(obj,c,s,t)
+%
+% Inputs:
+%   obj             MOxUnitTestResult instance.
+%   c               Single character representation of test case result;
+%                   Typically, the following characters are used:
+%                   'e' : error
+%                   'F' : failure
+%                   's' : skip
+%                   '.' : success
+%   t               MOxUnitTestCase instance with the test.
+%   dur             Duration of runtime for unit test (in seconds).
+%   reason          Handle to the error, or a string if no error.
+%
+% Output:
+%   x               The formatted XML report for this testcase.
+%
+% Notes:
+%   - This is a helper function for addError, addFailure, addSkip and
+%     addSuccess to add an XML report of a test result to the XML stack
+%     in the result object after a test has been run.
+%
+% See also: report, addError, addFailure, addSkip, addSuccess
+%
+% SCL 2015
+
+    % Set up the XML node for the testcase.
+    % They all start the same way.
+    x = '<testcase classname="%s" file="%s" name="%s" time="%.9f"';
+
+    % We need to include the classname although we don't rightly have
+    % a classname to give. Instead, we use the path to the file.
+    classpath = get(t,'location');
+    % Remove out the path to the testing directory
+    classpath = strrep(classpath, pwd, '');
```

Comment:
To the first, because it gives a better user experience and is more compatible with the way classname works for other JUnit XML outputs.

To the second, yes I should change it to only remove from the start. I will do that.